packages/@aws-cdk-testing/framework-integ/test/aws-s3-deployment/test/integ.bucket-deployment.ts

```diff
@@ -99,6 +99,19 @@ class TestBucketDeployment extends cdk.Stack {
       retainOnDelete: false, // default is true, which will block the integration test cleanup
     });
     deploy5.addSource(s3deploy.Source.data('some-key', 'helloworld'));
+
+    const bucket6 = new s3.Bucket(this, 'Destination6', {
+      publicReadAccess: false,
+      removalPolicy: cdk.RemovalPolicy.DESTROY,
+      autoDeleteObjects: true, // needed for integration test cleanup
+    });
+
+    new s3deploy.BucketDeployment(this, 'DeployMe6', {
+      sources: [s3deploy.Source.asset(path.join(__dirname, 'my-website-second'))],
+      destinationBucket: bucket6,
+      retainOnDelete: false, // default is true, which will block the integration test cleanup
+      outputObjectKeys: false,
+    });
```

Comment:
Can we confirm that this test fails with `outputObjectKeys = true`? That is, will this `BucketDeployment` make a response object that exceeds the 4K character count, but deploys successfully when we do not include the object keys in the output?

Reply:
I added a new integration Test case that test the large number of assets that should fail if the new property is not there